.github/workflows/dangerci.yml

```
@@ -20,18 +20,18 @@ jobs:
-                uses: actions/checkout@v2
+                uses: actions/checkout@v7
 
             -   name: Use NodeJS ${{ matrix.node-version }}
-                uses: actions/setup-node@v2
+                uses: actions/setup-node@v7
                 with:
                     node-version: ${{ matrix.node-version }}
 
             -   name: Get yarn cache directory path
                 id: cache-dir-path
                 run: echo "::set-output name=dir::$(yarn cache dir)"
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)